scripts/register-completions.bash

```diff
@@ -5,7 +5,11 @@ _dotnet_bash_complete()
   local word=${COMP_WORDS[COMP_CWORD]}
 
   local completions
-  completions="$(dotnet complete --position "${COMP_POINT}" "${COMP_LINE}")"
+  if dotnet complete >/dev/null 2>/dev/null ; then
```

Comment:
My only concern here is the perf impact of invoking `dotnet complete` twice.

Reply:
Could we instead just check `$?` for non-zero and return `""` in that case?

Reply:
We'd have to generally always stderr to `/dev/null` though.

Reply:
Good point. Does the updated patch look okay?